### ISSUE TITLE
0.23.0 breaks SumatraOpener

### ISSUE THREAD

**Issue by nscaife** [thomasjo/atom-latex#77]
On 0.23.0, the Sumatra opener no longer opens SumatraPDF.  I did a quick debug and it looks like the command is being constructed correctly and `child_process.exec` is being called.  It looks like the problem happens around `child_process.js:634`.  The args array is created like this:

```
file: "C:\Windows\system32\cmd.exe"
args[0]: "/s"
args[1]: "/c"
args[2]: ""C:\Program Files (x86)\SumatraPDF\SumatraPDF.exe -forward-search <mastertexfile> <line> <outputpdf>""
```

`cmd.exe` is parsing `args[2]` like it is the name of the executable that needs to run (instead of an executable and its arguments), and fails to start this long command.  We used to send in the argument list to `child_process.execFile`, which takes an array of inputs instead of a string -- apparently `exec` and `execFile` handle arguments a bit differently.

**Comment by nscaife** [comment]
A thought: `execFile` instead of `exec` makes a bit more sense to use on Windows, since it doesn't need to start a shell to start the PDF opener.

**Comment by thomasjo** [comment]
Damn, sorry about this. This is a dumb regression on my part. I'll get it fixed right away, and yes, I agree that using `execFile` is better. Thanks for the report and help with debugging :bow:

**Comment by thomasjo** [comment]
@nscaife My Windows VM seems to be broken. Would you mind checking out the `tj-fix-77` branch to see if that resolves the issue?

**Comment by nscaife** [comment]
@thomasjo It looks like this fixes it! Thanks for looking into it.

**Comment by thomasjo** [comment]
Great, thanks for testing! Just published v0.23.1.
